socorro/processor/hybrid_processor.py

```diff
@@ -310,7 +310,7 @@ def _temp_raw_crash_json_file(self, raw_crash, crash_id):
             os.unlink(file_pathname)
 
     #--------------------------------------------------------------------------
-    def convert_raw_crash_to_processed_crash(self, raw_crash, raw_dumps):
+    def process_crash(self, raw_crash, raw_dumps, ignored_processed_crash):
```

Comment:
this change represents the idea that the ```convert_raw_crash_to_processed_crash``` method name was no longer accurate.  The name ```process_crash``` covers both cases where raw is converted to processed, and processed is augmented by additional processing.